eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -246,6 +246,8 @@ object NodeParams {
         randomizeRouteSelection = config.getBoolean("router.randomize-route-selection"),
         requestNodeAnnouncements = config.getBoolean("router.sync.request-node-announcements"),
         encodingType = routerSyncEncodingType,
+        channelRangeChunkSize = config.getInt("router.sync.channel-range-chunk-size"),
```

Comment:
I think it's also good to add a require on the max value (like you did in the first commit), isn't it?

Reply:
Yres, but for that we have to agree on what are the actual max values, and it has to be maintained. Given that this is very low level, and that we strongly discourage users from changing it, I'm not sure it is worth it to change.

Reply:
Fair enough